java_fundamentals_block/optional_task/src/main/java/OptionalTask1.java

```diff
@@ -0,0 +1,82 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class OptionalTask1 {
+
+    private static BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+    public static int[] inputElem() throws IOException {
```

Comment:
i suggest renaming to getElementsArrayFromConsole

Reply:
good idea. Done